src/components/hoc/withAdapter.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React, {useContext} from 'react';
+import {create, act} from 'react-test-renderer';
+import waitForExpect from 'wait-for-expect';
+import PropTypes from 'prop-types';
+
+import withAdapter from './withAdapter';
+import WebexJSONAdapter from '../../adapters/WebexJSONAdapter';
+import {AdapterContext} from '../hooks';
+
+describe('withAdapter HOC', () => {
+  const adapterConnectedChanged = jest.fn();
+  const contextAdapterChanged = jest.fn();
+  const customPropsChanged = jest.fn();
+  let adapter;
+  let testRenderer;
+
+  // eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
Should probably add a description instead of disabling warning. Suggestions is incomplete tho
```suggestion
  /**
   * Sample component used for testing.
   */
```